kcidb/db/sqlite/schema.py

```diff
@@ -84,21 +97,32 @@ def __init__(self, constraint=None, pack=None, unpack=None):
             constraint:     The column's constraint.
                             A member of the Constraint enum, or None,
                             meaning no constraint.
-            pack:           The function packing the JSON representation of
-                            column's values into the SQLite representation.
-            unpack:         The function unpacking the SQLite representation
-                            of column's values into the JSON representation.
         """
         assert constraint is None or isinstance(constraint, Constraint)
         assert pack is None or callable(pack)
         assert unpack is None or callable(unpack)
```

Comment:
Shouldn't `pack` and `unpack` parameters be removed from the constructor? I also think the validation statement should be removed.